### DISCUSSION TITLE
Is there a way to sort Axis-Y values (Axis Left) as descending at Nivo/Bump?

### DISCUSSION THREAD

**Discussion by baris-finixio** [plouc/nivo#1922] [Q&A]
Hi guys,

Currently, at the Bump chart, values order as ascending, so the small (number) values are placed higher than bigger values.
Here is an example from Nivo's storybook: https://nivo.rocks/storybook/?path=/story/bump--default
As you can see in the example, on the left axis, the values start from "1" at the top.

Unfortunately, I couldn't find a way the re-order it. I need higher values on the top, very similar to an ordinary line chart. If I have to example my question over the given example above, I need "6" at the very top.

Thank you so much in advance.

**Comment by georgewritescode** [comment]
A workaround to reverse the order y axis is to multiply values of y-axis by -1, then add a formatter to y axis so appears positive in the chart itself. 

```
        axisLeft={{
            tickSize: 5,
            tickPadding: 5,
            tickRotation: 0,
            legend: 'ranking',
            legendPosition: 'middle',
            legendOffset: -40,
            truncateTickAt: 0,
            format: (v) => (v * -1)
        }}
